### PR TITLE
Add testing of final ii binary with BadMedicine.Dicom

### PR DESCRIPTION
**Pull request by tznind**

### CODE REVIEW

**Comment by tznind** [comment]
Ok well its passing... but says:

```
2022-07-22 11:27:19.9418| INFO|IsIdentifiable.Runners.DicomFileRunner|Processing '/home/runner/work/IsIdentifiable/IsIdentifiable/testdicoms/2004/10/15/2.25.239558545468471151818484951102659290560.dcm'||
2022-07-22 11:27:19.9418| INFO|IsIdentifiable.Runners.DicomFileRunner| Frame 0 in '/home/runner/work/IsIdentifiable/IsIdentifiable/testdicoms/2004/10/15/2.25.239558545468471151818484951102659290560.dcm'||
Error in pixWriteMemPng: function not present
Error in pixWriteMemPng: function not present
Error in pixWriteMemPng: function not present
```

Maybe we need some boot strapping of image formats?

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -79,6 +79,22 @@ jobs:
...
+         mkdir ./testii &&
+         cp ./release/ii-cli-linux-x64.tar.gz ./testii/ &&
+         cd ./testii &&
+         tar -xvf ./ii-cli-linux-x64.tar.gz &&
+         cd .. &&
+         ./testii/linux-x64/ii --help &&
+
+         wget https://github.com/HicServices/BadMedicine.Dicom/releases/download/v0.0.14/baddicom-linux-x64-v0.0.14.tar.gz &&
+         tar -xvf ./baddicom-linux-x64-v0.0.14.tar.gz &&
+         ./linux-x64/BadDicom -s 100 ./testdicoms/ 10 10 CT &&
```
Comment (rkm) [inline .github/workflows/dotnet-core.yml:93]: we don't need the `&&` at the end of each line here. Starting the whole block with `|` means that newlines are preserved: https://yaml-multiline.info/

**Review by rkm — CHANGES_REQUESTED** [review]

File: .github/workflows/dotnet-core.yml (hunk 1 above)
Comment (tznind) [reply .github/workflows/dotnet-core.yml:93]: Will it still report any nonzero exit codes as failure (for any step)?  I thought that was how you got linux to aggregate exit codes?

Comment (rkm) [reply .github/workflows/dotnet-core.yml:93]: I think in that case it's better to put `set -e` at the start of the script, which will cause an exit on the first non-zero return code. See [here](http://redsymbol.net/articles/unofficial-bash-strict-mode/).

**Review by rkm — APPROVED** [review]

**Comment by rkm** [comment]
Looks fine, though we should probably tweak the logging verbosity in the CI output in future

**Pull request merged**